src/test/java/org/jpeek/skeleton/SkeletonTest.java

```diff
@@ -142,6 +142,20 @@ public void findFieldWithQualifiedName() {
         );
     }
 
+    @Test
+    public void findSchemaOfSkeleton() {
+        MatcherAssert.assertThat(
+                XhtmlMatchers.xhtml(
+                        new Skeleton(
+                                new FakeBase(
```

Comment:
@stepanov-dmitry please fix the indentation here

Reply:
@paulodamaso thx, fixed. Do we have some code formatters for IDEs?